junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvSource.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.jupiter.params.provider;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ ElementType.ANNOTATION_TYPE, ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@ArgumentsSource(CsvArgumentsProvider.class)
+public @interface CsvSource {
+
+	String[] value();
+
+	char delimiter() default ',';
+
```

Comment:
What about the wrapping character (e.g., `"` or perhaps better defaulting to `'` since double quotes are used for Strings in Java annotations.) -- is that supported?